README.rst

```diff
@@ -31,7 +34,33 @@ Versioning
 ----------
 
 The TUF specification uses `Semantic Versioning 2.0.0 <https://semver.org/>`_
-for its version numbers.
+(semver) for its version numbers, and a gitflow-based release management:
+
+- The 'master' branch of this repository always points to the latest stable
+  version of the specification.
+- The 'draft' branch of this repository always points to the latest development
+  version of the specification and must always be based off of the latest
+  'master' branch.
+- Contributors must submit changes as pull requests against these branches,
+  depending on the type of the change (see semver rules).
+- For patch-type changes, pull requests may be submitted directly against the
+  'master' branch.
+- For major- and minor-type changes, pull requests must be submitted against
+  the 'draft' branch.
```

Comment:
It's slightly odd to me that the person sending a PR doesn't indicate minor vs major.  Also, it sounds like those are getting intertwined in a way we may not want.  What if we want to push a release with some backwards compat (minor version) changes but not others that are not backwards compatible?  Do we disentangle these changes at that time?

Reply:
I'd prefer to have a model where there are major, minor, and patch branches and they interrelate in the way your master and draft branches do.

I like the overall approach though!

Reply:
> It's slightly odd to me that the person sending a PR doesn't indicate minor vs major. 

I think the person who sends the PR should not be bothered with the question into what future release the submitted change goes, regardless of it being major, minor or patch. This is the task of the maintainer. Maybe I should add a note to this section that says maintainers may [change the target of a pending PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request)?

> Also, it sounds like those are getting intertwined in a way we may not want. What if we want to push a release with some backwards compat (minor version) changes but not others that are not backwards compatible? Do we disentangle these changes at that time?

Coordinating major and minor releases can still be done, either by holding back PRs from being merged into `draft`, or, if it's too late for that, by shuffling around the git history on `draft`. The consensus in #73 seemed to be that the operational simplicity of having one working branch, plus the option to directly patch `master`, outweighs the flexibility of multiple permanent draft branches.

> I like the overall approach though!

Thanks.

Reply:
I agree with @lukpueh, we want to make PRs as simple as possible. As long as we merge draft into master regularly, I don't think that we will need any additional branches.